Cpp/fostgres/sql.cpp

```diff
@@ -69,10 +69,11 @@ fostlib::json fostgres::connection_config(
                             "config", "current", config)("lookup", lookup);
                     loc.del_key(config);
                 } else {
-                    const auto envname =
+                    auto envname =
```

Comment:
Why this one doesn't be `const` anymore?

Reply:
Because of the `shrink_to_fit` on the following line. It's a mutating method that returns the `char const *`. In order to do that it may need to re-allocate the string.